steps/aws-secrets-manager/1.0.0/step.yml

```diff
@@ -0,0 +1,62 @@
+title: Fetch secrets
+summary: Fetch secrets from AWS Secrets Manager
+description: |
+  The main intention of this custom step is to allow storage of sensitive data on AWS Secrets Manager, and fetching them on-demand during Bitrise workflow execution.
+
+  When a workflow runs, the step fetches a list of specified secrets from AWS Secrets Manager.
+
+  The fetched secrets are then propagated into subsequent steps in the workflow.
+
+  Checkout the main repository README for usage guide.
```

Comment:
Please wire in inputs to get the required authentication credentials, so that the step clearly expresses, how the authentication can be done.

You can mark the Step Input as a [sensitive input](https://devcenter.bitrise.io/contributors/create-your-own-step/#sensitive-inputs) so that you enforce to use Bitrise secrets as value for the input.

Reply:
Addressed in moneylion/bitrise-step-aws-secrets-manager@18db9fda4e3bbeb88b47e75f4bb4cde7ca0ba1eb.